addons/mass_mailing/static/src/scss/mass_mailing.scss

```diff
@@ -80,16 +80,31 @@ body.editor_has_snippets .o_field_mass_mailing_html div.d-flex:has(iframe) {
         }
         &.o_kanban_ungrouped {
             padding: 0;
-            .o_mailing_list_kanban_grouped {
-                display: none !important;
-            }
             .o_kanban_record {
                 width: 100%;
                 margin: 0;
             }
         }
-        &.o_kanban_grouped .o_mailing_list_kanban_ungrouped {
-            display: none !important;
+        &.o_kanban_grouped {
+            .flex-row > div {
+                flex: 1 1 100% !important;
+                max-width: 100% !important;
+            }
+            .flex-row > div.col-6 {
+                flex: 1 1 50% !important;
+                max-width: 50% !important;
+            }
+            .flex-row > div.d-none {
+                display: none !important;
+            }
+            .o_mailing_list_kanban_stats > a {
+                font-size: 1.4rem !important;
+                height: 40px;
```

Comment:
![image](https://github.com/user-attachments/assets/13174932-b602-4efe-aca5-210b19e69513)

Fixed height can lead to issues if longer translations for some languages, would avoid if possible.

Reply:
that's what i referrred from ".o_mailing_list_kanban_stats > a { ... @include media-breakpoint-down(sm)" since that's the styling being used for breakpoint,  should i change it for both?

Reply:
Why not, if you find a concise and elegant solution.

Maybe we could keep the fixed height and add some text ellipsis if too long?
To investigate a bit.